docs/workflow/building/libraries/cross-building.md

```diff
@@ -24,108 +23,102 @@ and for arm64 you need:
+    artifacts/bin/native/net7.0-Linux-Release-arm/libSystem.IO.Compression.Native.a
+    artifacts/bin/native/net7.0-Linux-Release-arm/libSystem.IO.Compression.Native.so
+    artifacts/bin/native/net7.0-Linux-Release-arm/libSystem.IO.Compression.Native.so.dbg
+    artifacts/bin/native/net7.0-Linux-Release-arm/libSystem.IO.Ports.Native.a
+    artifacts/bin/native/net7.0-Linux-Release-arm/libSystem.IO.Ports.Native.so
+    artifacts/bin/native/net7.0-Linux-Release-arm/libSystem.IO.Ports.Native.so.dbg
+    artifacts/bin/native/net7.0-Linux-Release-arm/libSystem.Native.a
+    artifacts/bin/native/net7.0-Linux-Release-arm/libSystem.Native.so
+    artifacts/bin/native/net7.0-Linux-Release-arm/libSystem.Native.so.dbg
+    artifacts/bin/native/net7.0-Linux-Release-arm/libSystem.Net.Security.Native.a
+    artifacts/bin/native/net7.0-Linux-Release-arm/libSystem.Net.Security.Native.so
+    artifacts/bin/native/net7.0-Linux-Release-arm/libSystem.Net.Security.Native.so.dbg
+    artifacts/bin/native/net7.0-Linux-Release-arm/libSystem.Security.Cryptography.Native.OpenSsl.a
+    artifacts/bin/native/net7.0-Linux-Release-arm/libSystem.Security.Cryptography.Native.OpenSsl.so
+    artifacts/bin/native/net7.0-Linux-Release-arm/libSystem.Security.Cryptography.Native.OpenSsl.so.dbg
 
+    $ file artifacts/bin/native/net7.0-Linux-Release-arm/libSystem.Native.so
+    artifacts/bin/native/net7.0-Linux-Release-arm/libSystem.Native.so: ELF 32-bit LSB shared object, ARM, EABI5 version 1 (SYSV), dynamically linked, BuildID[sha1]=5f6f6f9c4012dffed133624867adf32ac2af130d, stripped
 
-Compiling for managed CoreFX
+
+Compile managed part of libraries
 ============================
-The managed components of CoreFX are architecture-independent and thus do not require a special build for arm, armel or arm64.
+The managed components of libraries are architecture-independent and, thus, do not require a special build for arm, armel or arm64 (this is true if ILLinker trimming is disabled with `/p:ILLinkTrimAssembly=false`).
 
 Many of the managed binaries are also OS-independent, e.g. System.Linq.dll, while some are OS-specific, e.g. System.IO.FileSystem.dll, with different builds for Windows and Linux.
 
-    $ ROOTFS_DIR=/mnt/corefx-cross/arm ./build.sh --arch arm
-
-You can also build just managed code with:
+Build of managed part of libraries requires presence of built native part of libraries.
 
-    $ ./build.sh --arch arm /p:BuildNative=false
+To build managed part of libraries for arm using subset (architecture-dependent, can't be used on other architectures):
 
-The output is at `artifacts/bin/[BuildSettings]` where `BuildSettings` looks something like `net5.0-<TargetOS>-Debug-<Architecture>`. Ex: `artifacts/bin/net5.0-Linux-Debug-x64`. For more details on the build configurations see [project-guidelines](/docs/coding-guidelines/project-guidelines.md)
+    $ ./build.sh --arch arm --librariesConfiguration Release --subset libs.sfx
```

Comment:
If you want to simplify the commands, you can pass in the subset as the first argument which then doesn't require the `--subset` named argument identifier.

```suggestion
    $ ./build.sh libs.sfx --arch arm --librariesConfiguration Release
```

Reply:
I've added a separate note about this form of build.sh invocation